feat(content) Embersylph Adjustments (#9711)
endless-sky/endless-sky: data/drak/indigenous.txt

	turret 0 0 "Absorption Organ"
	leak "blood" 2
	explode "blood" 1

> 	description "Embersylphs are strange creatures that glow in various hues, gracefully swinging several long appendiges in a strangely methodic movement. The only known behavior these creatures are known to display is an act of curiosity for other life."

Reviewer: ```suggestion
	description "Embersylphs are strange creatures that glow in various hues, gracefully swinging several long appendages in a strangely methodic movement. The only consistent behavior these creatures are known to display is curiosity for other life."
```

Minimum recommendation, removing redundancy and the odd "act of curiosity" phrasing in addition to spelling fix.

> 	description "Embersylphs are strange creatures that glow in various hues, gracefully swinging several long appendiges in a strangely methodic movement. The only known behavior these creatures are known to display is an act of curiosity for other life."

Reviewer: ```suggestion
	description "Embersylphs are strange creatures which glow in various hues while gracefully swinging several long appendages in a strangely methodic movement. The only consistent behavior these creatures are known to display is curiosity for other life."
```

Slightly more extensive style adjustment including the changes above.

> 	description "Embersylphs are strange creatures that glow in various hues, gracefully swinging several long appendiges in a strangely methodic movement. The only known behavior these creatures are known to display is an act of curiosity for other life."

Reviewer: ```suggestion
	description "Embersylphs are notably strange creatures, even compared to other known space fauna. Glowing in various hues while gracefully swinging several long appendages in a strangely methodical motion, the only consistent behavior these creatures are known to display is curiosity for other life."
```

Significantly more extensive style adjustment (and a bit more explicit comparison) including the changes above.

Could also go either way on shifting down just the "methodical motion" change to include in either of the other two options.
